.github/workflows/ci.yml

```
@@ -333,3 +333,25 @@ jobs:
         with:
           name: msrv
           path: msrv/
+
+  check-bevy-internal-imports-in-examples:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check for bevy_internal imports in examples
+        shell: bash
+        run: |
+          errors=""
+          for file in $(find examples -name '*.rs'); do
+              if grep -q "use bevy_internal" "$file"; then
+                  errors+="ERROR: Detected 'use bevy_internal' in $file\n"
+              fi
+          done
+          if [ -n "$errors" ]; then
+              echo "$errors"
```

```suggestion
              echo -e "$errors"
```

Fixes the line breaks